packages/transform-metering/test/test-transform.js

```diff
@@ -44,6 +44,7 @@ test('meter transform', async t => {
       );
       const rewritten = await fs.promises
         .readFile(`${base}/${testDir}/rewrite.js`, 'utf8')
+        .then(s => s.replace(/(\r\n|\r)/g, '\n'))
```

Comment:
this seems out of place in this PR. Is it updating the newline characters for some reason?

Reply:
Whoops, saw your description: Also fixes one test to be insensitive to line ends.

Reply:
It is a test-only change to make the test work independent of line numbers. It was generated in pairing with @michaelfig. If it was easy to drop that change here, I would consider separating it.